frontend/src/components/prompts/Info.vue

```diff
@@ -166,6 +175,26 @@ export default {
         this.$showError(e);
       }
     },
+    foldersize: async function (event) {
```

Comment:
```suggestion
    folderSize: async function (event) {
```

Reply:
Thank you for the patient code review. The function name filesize() was in my mind when naming these functions, should I change filesize() to fileSize() also to make them conformant?